examples/resources/metal_connection/shared_nimf_to_csp.tf

```diff
@@ -6,16 +6,13 @@ resource "equinix_metal_connection" "example" {
   name            = "tf-port-to-metal-legacy"
   project_id      = local.my_project_id
   metro           = "SV"
-  redundancy      = "redundant"
-  type            = "shared"
+  redundancy      = "primary"
+  type            = "shared_port_vlan"
   contact_email   = "username@example.com"
-  vlans              = [
-    equinix_metal_vlan.example1.vxlan,
-    equinix_metal_vlan.example2.vxlan
-  ]
+  vlans              = [ equinix_metal_vlan.example1.vxlan ]
 }
 data "equinix_fabric_service_profiles" "zside" {
-  count = var.zside_ap_type == "SP" ? 1 : 0
+  count = local.zside_ap_type == "SP" ? 1 : 0
```

Comment:
This condition causes the example to fail when converting to Pulumi and does not provide any value here as this example is just for connecting to a service provider. It would be helpful if you removed the `count` condition here. This update will also require updating the line referencing it `uuid = data.equinix_fabric_service_profiles.zside[0].id` with `uuid = data.equinix_fabric_service_profiles.zside.id`.